tests/test_sirius.py

```
@@ -47,3 +53,33 @@ def test_initialize(
     assert sirius.artist == "Artist"
     assert sirius.song == "Song"
     assert sirius.chname == "ChName"
+    assert sirius.searchmode == SiriusSearchMode.PRESET
+
+
+def test_searchmode(
+    connection: YncaConnectionMock, update_callback: Callable[[str, Any], None]
```

Ruff (0.15.4)

[warning] Unused function argument: `update_callback`

(ARG001)